NaReguaApp/src/contexts/auth.js

```diff
@@ -0,0 +1,64 @@
+import React, { createContext, useState, useEffect, useContext } from 'react';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import api from '../services/api';
+
+const AuthContext = createContext({});
+
+export const AuthProvider = ({ children }) => {
+	const [user, setUser] = useState(null);
+	const [isProvider, setIsProvider] = useState(false);
```

Comment:
Codacy has a fix for the issue: [Replace `↹` with `··`](https://app.codacy.com/gh/UFAPE-Projetao20202/naregua_front/pullRequest?prid=8386487)
```suggestion
  const [isProvider, setIsProvider] = useState(false);
```